node/pkg/watchers/solana/client.go

```diff
@@ -404,6 +406,12 @@ func (s *SolanaWatcher) Run(ctx context.Context) error {
 					return err
 				}
 			case m := <-s.obsvReqC:
+				if m.ChainId > math.MaxUint16 {
+					logger.Error("chain id for observation request is not a valid uint16",
```

Comment:
does this need a continue or something afterwards? I think this just logs an error and then proceeds to use the value anyway 🤔

Reply:
Updated, agreed `continue` makes sense here